core/src/main/java/org/geysermc/geyser/entity/type/TextDisplayEntity.java

```diff
@@ -37,13 +37,16 @@
 
 // Note: 1.19.4 requires that the billboard is set to something in order to show, on Java Edition
 public class TextDisplayEntity extends DisplayBaseEntity {
+
+    private Component text;
+
     public TextDisplayEntity(GeyserSession session, int entityId, long geyserId, UUID uuid, EntityDefinition<?> definition, Vector3f position, Vector3f motion, float yaw, float pitch, float headYaw) {
-        super(session, entityId, geyserId, uuid, definition, position.add(0, definition.offset(), 0), motion, yaw, pitch, headYaw);
+        super(session, entityId, geyserId, uuid, definition, position.add(0, definition.offset() + 20, 0), motion, yaw, pitch, headYaw);
```

Comment:
Can we store this magic 20 number somewhere since its used 2x and also give it an explanation so its not so magic?

Reply:
A y offset of 20 blocks here seems wrong too.. further, this is applied once during spawn without a location update, so if it's not completely off, this isn't necessary at all

Reply:
Yes sorry I forgot to remove that while I was testing